src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
Well EventsEmitter is an abstract class used to extends and to provided a common pubsub pattern. I'm no extending a class with many behaviours. It's an "event dispatching" class. It's more a trait than a class.